### PR TITLE
apps: add defaults for deployment id and component name

### PR DESCRIPTION
**Pull request by bouk**
This defaults the deployment ID to the current deployment, and the component name to '' so aggregate logs are used.

APPS-1547

### CODE REVIEW

File: commands/apps.go [hunk 1]
```diff
@@ -328,12 +329,26 @@ func RunAppsListDeployments(c *CmdConfig) error {
...
 // RunAppsGetLogs gets app logs for a given component.
 func RunAppsGetLogs(c *CmdConfig) error {
-	if len(c.Args) < 3 {
+	if len(c.Args) < 1 {
 		return doctl.NewMissingArgsErr(c.NS)
 	}
 	appID := c.Args[0]
-	deploymentID := c.Args[1]
-	component := c.Args[2]
+	component := "web"
```
Comment (snormore) [inline commands/apps.go:336]: Should we default this to the first component in the active deployment instead?

Comment (bouk) [reply commands/apps.go:336]: Yes, that would be better.

**Comment by kamaln7** [comment]
I think we should default to aggregate logs if a component name isn’t set

**Review by snormore — DISMISSED** [review]

**Comment by bouk** [comment]
I've updated it to use godo 1.44

**Review by snormore — APPROVED** [review]

**Review by nicktate — APPROVED** [review]

**Review by andrewsomething — APPROVED** [review]
:+1: LGTM!

**Pull request merged**